### PR TITLE
[zh] Sync Configuring failover for external services blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate Configuring failover for external services blog into Chinese:

```
content/zh/blog/2021/external-locality-failover/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+该功能就可以派上用场。根据工作负载运行的位置，通常可以有多个端点供用户选择。
+例如，Amazon 的 DynamoDB 就可以提供跨区域的[多个端点](https://docs.aws.amazon.com/general/latest/gr/ddb.html)。
+由于延迟原因，常常希望选择使用离您工作负载最近的端点，
+但是为了防止未按照预期正常工作的情况发生，您可能需要配置自动故障转移使流量可以指向另一个端点。
+
+与在服务网格内运行的服务类似，您可以对 Istio
+进行异常检测配置来实现将流量指向健康端点的故障转移操作，
+并且该配置仍然对您的应用程序完全透明。在此示例中，我们将使用
+Amazon DynamoDB 端点选择与 Google Kubernetes
+Engine（GKE）集群中运行的工作负载处于相同或接近的主要区域。
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:21]:
```suggestion
Amazon DynamoDB 端点选择与 Google Kubernetes Engine（GKE
集群中运行的工作负载处于相同或接近的主要区域。
```

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+endpoints:
+- address: dynamodb.us-east-1.amazonaws.com
+  locality: us-east1
+  ports:
+    http: 80
+- address: dynamodb.us-west-1.amazonaws.com
+  locality: us-west
+  ports:
+    http: 80
+{{< /text >}}
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:72]: The indentations are incorrect, and inconsistent with en upstream now. What's happened?

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+您将看到 Istio 会分别向两个端点都发送请求。
+但是我们只希望请求被发送到与我们节点标记为相同区域的端点中。
+
+为此，我们需要配置一个 `DestinationRule`。
+
+## 使用 `DestinationRule` 设置故障转移条件 {#set-failover-conditions-using-a-destinationrule}
+
+Istio 的 `DestinationRule` 允许您配置负载均衡、连接池和异常检测设置。
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:98]:
```suggestion
Istio 的 `DestinationRule` 允许您配置负载均衡、连接池和离群检测设置。
```

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+metadata:
+name: mydynamodb
+spec:
+host: mydb.com
+trafficPolicy:
+  outlierDetection:
+    consecutive5xxErrors: 1
+    interval: 15s
+    baseEjectionTime: 1m
+{{< /text >}}
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:113]: same issue about indentations

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+endpoints:
+- address: dynamodb.us-east-1.amazonaws.com
+  locality: us-east1
+  ports:
+    http: 81 # INVALID - This is purposefully wrong to trigger failover
+- address: dynamodb.us-west-1.amazonaws.com
+  locality: us-west
+  ports:
+    http: 80
+{{< /text >}}
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:159]: same issues about indentations

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+  name: http-port
+  protocol: HTTP
+  targetPort: 443
+resolution: DNS
+endpoints:
+- address: dynamodb.us-east-1.amazonaws.com
+  locality: us-east1
+- address: dynamodb.us-west-1.amazonaws.com
+  locality: us-west
+{{< /text >}}
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:206]: same issues about indentations

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 7]
```diff
@@ -0,0 +1,242 @@
...
+    localityLbSetting:
+      enabled: true
+      failover:
+        - from: us-east1
+          to: us-west
+  outlierDetection:
+    consecutive5xxErrors: 1
+    interval: 15s
+    baseEjectionTime: 1m
+{{< /text >}}
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:232]: check the indentations pls

File: content/zh/blog/2021/external-locality-failover/index.md [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+
+`DestinationRule` 现在执行 TLS 源并配置异常检测。
```
Comment (windsonsea) [inline content/zh/blog/2021/external-locality-failover/index.md:234]:
```suggestion
`DestinationRule` 现在执行 TLS 源并配置离群检测。
```

**Review by windsonsea — COMMENTED** [review]
All YAML code blocks should be double-checked for indentation.

**Comment by wilsonwu** [comment]
Thanks I found the en version also has some problem, I will update ASAP and fix the en version later.

**Comment by windsonsea** [comment]
lol, see #13424

**Review by windsonsea — APPROVED** [review]
Thanks. LGTM though we can focus more on newer blogs.

**Comment by wilsonwu** [comment]
> Thanks. LGTM though we can focus more on newer blogs.

Yes, I will continuing translate new blogs, and also sync some useful old ones.

**Pull request merged**